tests/robot-tests/tests/general_public/table_tool_exclusions_by_geographic_level.robot

```diff
@@ -101,22 +101,21 @@ Validate Bury Number of fixed period exclusions row
 
 User generates a permanent link
     [Tags]   HappyPath
-    user waits until page contains button  Share your table  60
-    user clicks button    Share your table
+    user waits until page contains button  Generate shareable link  60
+    user clicks button    Generate shareable link
     user waits until page contains testid  permalink-generated-url
     user checks generated permalink is valid
 
 User validates permanent link works correctly
     [Tags]   HappyPath
     user clicks link   View share link
-    select window    NEW
+    switch window   locator=NEW
```

Comment:
Can use the new `user selects newly opened window` keyword you've created here.

Reply:
Noice